hutch_python/load_conf.py

```diff
@@ -157,6 +158,8 @@ def load_conf(conf, hutch_dir=None):
 
     # Make RunEngine
     RE = RunEngine({})
+    bec = BestEffortCallback()
+    RE.subscribe(bec)
```

Comment:
Should we also install the `qt_kicker` here?

Reply:
Good catch

Reply:
If you want it to pass:
```
before_script:
    #Take from docs.travis-ci.com/user/gui-and-headless-browsers
  - "export DISPLAY=:99.0"
  - "sh -e /etc/init.d/xvfb start"
  - sleep 3 # give xvfb some time to start
```
Otherwise the try/except works